apps/speed.c

```diff
@@ -1778,7 +1778,13 @@ int speed_main(int argc, char **argv)
         buflen = lengths[size_num - 1];
         if (buflen < 36)    /* size of random vector in RSA benchmark */
             buflen = 36;
-        buflen += MAX_MISALIGNMENT + 1;
+        if (INT_MAX - (MAX_MISALIGNMENT + 1) < buflen)
+        {
```

Comment:
Placement of `{` should be on the end of the preceding line

Reply:
changed this.